llvm/include/llvm/Analysis/LazyCallGraph.h

```diff
@@ -1308,6 +1309,10 @@ class LazyCallGraphDOTPrinterPass
   static bool isRequired() { return true; }
 };
 
+#ifdef _WIN32
```

Comment:
Why are these Windows specific? Can we not universally have these be homed on all platforms?

Reply:
I wasn't sure if i would be changing the existing behaviour for Any Ids on non windows platforms in some unexpected way